frontend/packages/dev-console/integration-tests/support/step-definitions/quck-starts/filter-quick-starts-catalog.ts

```diff
@@ -0,0 +1,93 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+import { quickStartsPO } from '../../pageObjects';
+import { quickStartsPage } from '../../pages';
+
+When('user clicks on "View all quick starts" on Build with guided documentation card', () => {
+  quickStartsPage.quickStartsCatalog();
+});
+
+Then('user can see Quick Starts catalog page', () => {
+  cy.get(quickStartsPO.quickStartTitle).should('be.visible');
+});
+
+Then('user can see filter by keyword search bar', () => {
+  cy.get(quickStartsPO.filterKeyword).should('be.visible');
+});
+
+Then('user can see Status filter dropdown', () => {
+  cy.get(quickStartsPO.statusFilter).should('be.visible');
+});
+
+Given('user is at Quick Starts catalog page', () => {
+  quickStartsPage.quickStartsCatalog();
+});
+
+When('user clicks on filter by keyword search bar', () => {
+  cy.get(quickStartsPO.filterKeyword).click();
+});
+
+When('user enters {string}', (filterName: string) => {
+  quickStartsPage.filterByKeyword(filterName);
+});
+
+Then('user can see "Install the OpenShift Pipelines Operator" Quick Start is present', () => {
```

Comment:
Can we make this call dynamic?

```suggestion
Then('user can see {string} Quick Start', (quickStartName: string) => {
```

Reply:
Although we use static page objects, I've made it dynamic